packages/harmony/src/storybook/theme/colors.ts

```diff
@@ -0,0 +1,22 @@
+export const primitives = {
+  primary: {
+    V050: '#F7F6FC',
+    V100: '#F1EFFF',
+    V200: '#C7BFFD',
+    V300: '#907FFB',
+    V400: '#5A49CA',
+    V500: '#2F2668',
+    V600: '#201C3D'
+  },
+  neutral: {
+    N050: '#FFFFFF',
```

Comment:
neutral 50 is white? probably on spec, but seems odd

Reply:
yeah odd for sure...